llvm-runtimes/libcxx/libcxx-22.0.0.9999.ebuild

```
@@ -191,6 +191,20 @@ multilib_src_install() {
 		dolib.so lib/libc++_shared.so
 		use static-libs && dolib.a lib/libc++_static.a
 	fi
+
+	local install_prefix
```

```suggestion
	local install_prefix=
```

Otherwise, host environment may leak in.

---

Thanks